File: ExamenParcial_Material/Librerias/Clase 4/ejercicio01/principales/Principal.java
package principales;
import matematicas.Operacion;
class Principal{
	public static void main(String[] args){
		Operacion op = new Operacion();
		int a = 10;
		int b = 5;
		int c = op.sumar(a,b);
		int d = op.restar(a,b);
		System.out.println("El valor de a es: " + a);
		System.out.println("El valor de b es: " + b);
		System.out.println("La suma es: " + c);
		System.out.println("La resta es: " + d);
	}
}

File: ExamenParcial_Material/Librerias/ejercicio03/principales/Principal.cs
using Matematicas;
using System;
namespace Principales{
	class Principal{
		public static void Main(string[] args){
			Operacion op = new Operacion();
			int a = 10;
			int b = 5;
			int c = op.sumar(a,b);
			int d = op.restar(a,b);
			Console.WriteLine("El valor de a es: " + a);
			Console.WriteLine("El valor de b es: " + b);
			Console.WriteLine("La suma es: " + c);
			Console.WriteLine("La resta es: " + d);
		}
	}
}